TypeDefinition/GMS/GMSSensor.hpp

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2022 (c) Alen Galinec
+ */
+
+#pragma once
+#include <cstdint>
+#include <string>
+#include <vector>
+
+#include "../MachineTool/Tool.hpp"
+#include "../TypeDefinition.hpp"
+#include "../ns0/MultiStateDiscreteType.hpp"
+#include "Constants.hpp"
+
+namespace GMS {
+
+struct GMSToolLife_t : public machineTool::ToolLife_t<double> {
```

Comment:
Is this necessary?
I think we use in GMS also the machineTool ToolLifeType

Reply:
I can't remember why I did this. I have to check what happens when I use the MT ToolLife_t directly.

Reply:
You are right, the extra structure isn't necessary 👍 Removed it.